From 10b7f50fdfd5c0ab59c4ebbc855c58a2022bce38 Mon Sep 17 00:00:00 2001
From: Sun Yijin <yijin.sun@intel.com>
Date: Tue, 10 Jul 2012 17:31:59 +0800
Subject: [PORT FROM MAIN] Generate gzip package of system.img

BZ: 46708

[BOOT CAMP SPECIFIC]

Change-Id: Ib1edc6bb6bfa1df2bb413d001c2bc1d5535fc20c
Orig-Change-Id: I461fba34f13e6717dd17a2cfb2576ae5632d391c
Reviewed-on: http://android.intel.com:8080/56340
Reviewed-by: Beare, Bruce J <bruce.j.beare@intel.com>
Reviewed-by: Li, XiaojingX <xiaojingx.li@intel.com>
Tested-by: Li, XiaojingX <xiaojingx.li@intel.com>
Reviewed-on: http://android.intel.com:8080/74479
Reviewed-by: Yao, Yong <yong.yao@intel.com>
---
 core/Makefile |   12 ++++++++++++
 1 files changed, 12 insertions(+), 0 deletions(-)

diff --git a/core/Makefile b/core/Makefile
index 55ab6a5..ed85977 100644
--- a/core/Makefile
+++ b/core/Makefile
@@ -835,6 +835,18 @@ $(warning Warning: with dexpreopt enabled, you may need a full rebuild.)
 endif
 endif
 
+# -----------------------------------------------------------------
+# Zip package of system.img
+#
+INSTALLED_SYSTEMIMG_GZ_TARGET := $(PRODUCT_OUT)/system.img.gz
+
+$(INSTALLED_SYSTEMIMG_GZ_TARGET) : $(INSTALLED_SYSTEMIMAGE) | $(MINIGZIP)
+	@echo "Generate system.img.gz"
+	$(MINIGZIP) <$? >$@
+
+.PHONY: systemimg_gz
+systemimg_gz: $(INSTALLED_SYSTEMIMG_GZ_TARGET)
+
 #######
 ## system tarball
 define build-systemtarball-target
-- 
1.7.4.1

